android/src/main/java/com/pdftron/pdftronflutter/FlutterDocumentActivity.java

```diff
@@ -187,6 +222,13 @@ public boolean onOpenDocError() {
         return PluginUtils.handleOpenDocError(this);
     }
 
+    @Override
+    public void onNavButtonPressed() {
+        handleLeadingNavButtonPressed(this);
```

Comment:
don't call super here otherwise the activity will just close

Reply:
Sorry for missing this...